### PR TITLE
Add support for default json parameters

### PR DESCRIPTION
**Pull request by xubury**
Hi!
The default json file makes user run program in target's parent path which usually isn't cwd (when run_dir is nil).
Some people, like me, would perfer running program from cwd.
So I make a commit that allow user to change "default parameters" of that json file. 
What do you think?

### CODE REVIEW

**Comment by Shatur** [comment]
Okay, makes sense!

**Comment by Shatur** [comment]
Could you update the README.md too?

**Comment by xubury** [comment]
The README is updated now.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ require('cmake').setup({
   cmake_executable = 'cmake', -- CMake executable to run.
   save_before_build = true, -- Save all buffers before building.
   parameters_file = 'neovim.json', -- JSON file to store information about selected target, run arguments and build type.
+  default_parameters = { run_dir = '', args = {}, build_type = '' }, -- The default values in `parameters_file`.
```
Comment (Shatur) [inline README.md:57]: Default build type was `Debug`. Could you keep it here and in `lua/cmake/config.lua`?

**Review by Shatur — APPROVED** [review]

**Pull request merged**